puzzle/plugins/vcf.py

```diff
@@ -248,9 +248,10 @@ def variant(self, case_id, variant_id):
             Returns:
                 variant (Variant): The variant object for the given id
         """
-        for variant in self.variants(case_id):
-            if variant['variant_id'] == variant_id:
-                return variant
+        for variant_obj in self.variants(case_id, count=float('inf')):
```

Comment:
This fixes the issue @dnil mention about variants not loading after "load more" (hahaha)

The skip/limit stuff was in the way :P